### ISSUE TITLE
offline detection and mapping improvements 7

### ISSUE THREAD

**Issue by fneitzel** [pupil-labs/pupil#858]
In the moment it is possible to map the calibration to one range.
Defining more than one mapping range for a calibration section would be a nice feature.

**Comment by papr** [comment]
That would mean that the total mapping range would have holes. What is the benefit of that. This feature would also add much more complexity to the calibration section ui. The current way to implement this would be to add a second calibration section with the same calibration range but a different mapping range.

**Comment by fneitzel** [comment]
I thought of setting natural features for a distinct calibration. So i can have more than one natural features Section with different marker positions and as you mentioned I can use the holes for other calibrations.

Background is that i want to make two calibrations - one for far distance (on tv) - one for short distance (on phone) - so i use two different marker with different size, since i noticed that a smaller marker in short distance has more accuracy than a larger calibration marker (for distances). If the marker was not well detected (mostly because of the lighting situation) I want to use natural features.

Basically I know that this should work with one 3D binocular calibration. But I want to check if the are differences in gaze accuracy ... and also checking for differences if the headset moved slightly away with 3D Model while recording. So I two offline calibrations at the end of the video.

**Comment by willpatera** [comment]
@fneitzel You can already have multiple calibrations and set the calibration ranges accordingly.
